src/Symfony/Component/Notifier/Bridge/Bandwidth/BandwidthOptions.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Bandwidth;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author gnito-org <https://github.com/gnito-org>
+ */
+final class BandwidthOptions implements MessageOptionsInterface
+{
+    private array $options;
+
+    public function __construct(array $options = [])
+    {
+        $this->options = $options;
+    }
+
+    public function getAccountId(): ?string
+    {
+        return $this->options['account_id'] ?? null;
+    }
+
+    public function getApplicationId(): ?string
+    {
+        return $this->options['application_id'] ?? null;
+    }
+
+    public function getExpiration(): ?string
+    {
+        return $this->options['expiration'] ?? null;
+    }
+
+    public function getFrom(): ?string
+    {
+        return $this->options['from'] ?? null;
+    }
+
+    public function getMedia(): ?array
+    {
+        return $this->options['media'] ?? null;
+    }
+
+    public function getPriority(): ?string
+    {
+        return $this->options['priority'] ?? null;
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return $this->options['recipient_id'] ?? null;
+    }
+
+    public function getTag(): ?string
+    {
+        return $this->options['tag'] ?? null;
+    }
+
+    public function getTo(): ?array
+    {
+        return $this->options['to'] ?? null;
+    }
+
+    public function setAccountId(string $accountId): self
+    {
+        $this->options['account_id'] = $accountId;
+        return $this;
```

Comment:
```suggestion

        return $this;
```

Same for the other methods and PR's 😄